DFC.Api.Content/Function/Execute.cs

```diff
@@ -26,9 +26,9 @@ public class Execute
         private readonly IOptionsMonitor<ContentTypeSettings> _contentTypeSettings;
         private readonly IGraphDatabase _graphDatabase;
         private readonly IJsonFormatHelper _jsonFormatHelper;
```

Comment:
Can you check you naming conventions please, as "_" is not permitted using stylecop and cude ananlysis rulesets, think sonar cloud will reject it too